compel/arch/aarch64/src/lib/infect.c

```diff
@@ -83,6 +83,11 @@ int compel_get_task_regs(pid_t pid, user_regs_struct_t *regs, user_fpregs_struct
 	struct iovec iov;
 	int ret;
 
+	struct iovec gcs_iov = {
+			.iov_base = &fpsimd->gcs,
+			.iov_len = sizeof(fpsimd->gcs),
+	};
+
```

Comment:
why do we need a separate gcs_iov and cannot reuse iov as other PTRACE_GERREGSET calls?